PowerShell Contributing.md

```diff
@@ -0,0 +1,479 @@
+> 
+> Optional
+> 
+> - MS Teams
+> - A headset
+> - A camera
+
+Every month, there is a call with PowerShell Team members and community members are invited! 
+You can find the join link, as well as transcripts/youtube recordings of previous calls here: 
+<https://github.com/PowerShell/PowerShell-RFC/tree/master/CommunityCall/>
+
+Goal of the meeting is sharing roadmap/release info, and discussing RFC's.
+You can join in, ask questions and let your voice be heard.
+For example, if there is an RFC you have commented on, you can explain your view.
+
+### 5. Suggest an improvement
+
+> Requires
+> 
+> - GitHub account
+
+You can also come up with ideas yourself!
+
+If you have a cool improvement for a Cmdlet, a new feature or just a great idea, you can submit it as a [feature request](https://github.com/PowerShell/PowerShell/issues/new?assignees=&labels=Issue-Enhancement&template=Feature_Request.md&title=Feature+Request).
+
+The maintainers will look at your idea and let you know what they think.
+All genuine feature requests are handled with respect.
+So don't be afraid to share your thoughts! 
+
+Do take the time to find out if your idea has been proposed before and what was decided and why.
+You can filter on the label 'Issue-Enhancement'.
+
+In some cases it may be a good move to 'test' your idea against a few people first, before taking the time to fully explain what you have in mind in an issue, a great place to do so would be the [PowerShell Discord/Slack/IRC channel](https://poshcode.org).
+
+### 6. Contribute Help content
+
+> Requires
+> 
+> - GitHub account
+> - Knowledge of Markdown
+> 
+> Optional
+> 
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+
+You can contribute by writing, improving, or expanding help files for Get-Help.
+In particular, more examples for cmdlets are always welcome.
+
+Head over to the <https://github.com/MicrosoftDocs/Powershell-Docs> repo, and read their [contributing manual](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) to get started.
+
+> Tip: PowerShell Help is maintained for PowerShell versions 5.1 and higher. You may be asked to update help files for more than one version. 
+
+To do small contributions like correcting typos, just navigate to that file, click the 'Edit' button, and GitHub will create a copy (fork) on your own GitHub for you.
+When you are satisfied with your changes, you can save them, and submit a pull request to the _staging_ branch of the PowerShell-Docs repository.
+
+For bigger contributions, you are better off cloning your fork locally to your computer.
+This enables you to use VSCode and other editing tools.
+A guide on how to do that can be found [here](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/1-GET-STARTED.md#making-major-edits-to-existing-topics).
+
+With more examples, less typos, clearer wording, PowerShell can be easier to learn and use for everyone!
```

Comment:
Precede the last item of the summary with "and" instead of a comma.

Reply:
:+1: